packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st

```diff
@@ -0,0 +1,10 @@
+unit-tests
+testInspectDocument
+	
+	| possibleMorphs |
+	self editorModel inspectDocument.
+	possibleMorphs := Project current world submorphs.
+	possibleMorphs := possibleMorphs select: [: morph|  morph class = PluggableSystemWindow].
```

Comment:
I would style it like this. I think there is also one space too much after the |
(applies to the following lines as well.
```suggestion
	possibleMorphs := possibleMorphs select: [:morph | morph class = PluggableSystemWindow].
```

Reply:
fixed